README.md

```diff
@@ -4,4 +4,19 @@ The proto files can be consumed as GIT submodule or copied over and built direct
 
 The compiled files are published to central repositories (Maven, NPM...) from OpenTelemetry client libraries.
 
-See [contribution guidelines](CONTRIBUTING.md) if you would like to make any changes.
\ No newline at end of file
+See [contribution guidelines](CONTRIBUTING.md) if you would like to make any changes.
+
+## Maturity Level
+
+Component                | Maturity |
+-------------------------|----------|
+collector/metrics/*      | Alpha    |
+collector/trace/*        | Beta     |
+common/*                 | Beta     |
+metrics/*                | Alpha    |
+resource/*               | Beta     |
+trace/trace.proto        | Beta     |
+trace/trace_config.proto | Alpha    |
+
+(See [maturity-matrix.yaml](https://github.com/open-telemetry/community/blob/47813530864b9fe5a5146f466a58bd2bb94edc72/maturity-matrix.yaml#L57)
```

Comment:
should this be described in a yaml file though?

Reply:
I wanted this to be visible and easily readable. Happy to update the yaml file in the community repo too, which currently does not currently have an entry for the protocol (I can add).